### DISCUSSION TITLE
Got rugpulled

### DISCUSSION THREAD

**Discussion by crypt0mist** [blockscout/blockscout#11562] [Support]
can i get my domain nft back? coz i can see it in my wallet but cant make changes on it now after the rug pull

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.

**Reply by crypt0mist** [reply]
oh sorry 
do you know where i can contact?
